### ISSUE TITLE
Logs tailing freezes at midnight

### ISSUE THREAD

**Issue by hitchcott** [getumbrel/umbrel#361]
My umbrel node is currently syncing and I am checking the logs stream under http://umbrel.local/logs/

For the last couple of nights, I've woken up to the logs not streaming properly - this is fixed by restarting the node.

Each night, the logs stop streaming with:

```
Dec 26 00:00:19 umbrel electrs[728]: WARN - wait until IBD is over: headers=662977 blocks=476907 progress=0.4057944985507425
Dec 26 00:00:19 umbrel systemd[1]: Starting Daily man-db regeneration...
Dec 26 00:00:19 umbrel systemd[1]: Starting Rotate log files...
````

**Comment by lukechilds** [comment]
Yeah this is a known issue and it's really annoying.

You can still manually check the Umbrel logs via SSHing in and tailing the files in `~/umbrel/logs/` for the host daemon logs or running `docker-compose logs -f` for the container logs.

We haven't had a chance to look into why logrotate is killing syslog yet but if you find out please let us know!

**Comment by BTCFranky** [comment]
There is already frontrail v.4.9.2 available https://github.com/mthenw/frontail/releases
Curtrently Umbrel uses v4.9.1
Maybe this will fix this issue.

**Comment by AaronDewes** [comment]
> There is already frontrail v.4.9.2 available https://github.com/mthenw/frontail/releases
> Curtrently Umbrel uses v4.9.1
> Maybe this will fix this issue.

No, it doesn't, the issue isn't with frontail, but with the logs itself.

**Comment by STRML** [comment]
Issue is that you're only mounting `/var/log/syslog`, rather than `/var/log`. `tail -F` usually does the right thing here, but I think the volume mount is actually following the inode rather than the file name.

**Comment by alex27riva** [comment]
I'm currently having this issue on my Umbrel node.

**Comment by Sauron77** [comment]
I am also the same. Since Mar 19 00:00:39 umbrel systemd [1]: Man-db daily regeneration started.

**Comment by AaronDewes** [comment]
Please stop commenting about this, we have fixed the issue in 0.3.8 (not released yet)

**Comment by mayankchhabra** [comment]
Sorry, @alex27riva, for the issue. Like Aaron said, this should be fixed in the next update!
